dj_rest_auth/tests/requirements.pip

```diff
@@ -1,7 +1,7 @@
 coveralls==1.11.1
 django-allauth==0.61.1
-django>=2.2,<5.0
-djangorestframework-simplejwt==4.6.0
+django>=2.2
+djangorestframework-simplejwt>=5.3.1
```

Comment:
https://github.com/jazzband/djangorestframework-simplejwt/issues/756
without this
```shell
ImportError: Could not import 'dj_rest_auth.jwt_auth.JWTCookieAuthentication' for API setting 'DEFAULT_AUTHENTICATION_CLASSES'. ImportError: cannot import name 'utc' from 'django.utils.timezone'
```